doc/en/changelog.rst

```diff
@@ -134,25 +134,13 @@ Breaking Changes
 Deprecations
 ------------
 
-- `#6981 <https://github.com/pytest-dev/pytest/issues/6981>`_: Deprecate the ``pytest.collect`` module as it's just aliases into ``pytest``.
```

Comment:
These two will be reverted, I saved the text in https://github.com/pytest-dev/pytest/issues/7480#issuecomment-657106475 so we can bring them back later.